### PR TITLE
Rewriting version control section

### PR DESCRIPTION
**Pull request by gvwilson**
Probably easiest to read the new text straight up, rather than interpreting it as a diff.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -665,97 +665,133 @@ they keep track of what was changed in a file when and by whom,
...
+    and *not* have to spend days remembering what you did last
+    or figuring out what your collaborators have changed while you have been gone.
+2.  It should aid *fixability*:
+    having access to every version of code, figures, and data helps you figure out
+    why Figure 4 looks different now than it did last week.
+3.  It should support *sharing and collaboration*,
+    particularly by managing the process of merging independent changes made by different people,
+    and distributing those changes back to everyone in a controlled, traceable way.
+
+The "by hand" approach has three steps.
```
Comment (lexnederbragt) [inline index.md]: I'd more clearly delineate the two approaches (perhaps even with another subheading but that may be overkill). E.g.

'Our first suggested, "by hand" approach has three steps.'

File: index.md [hunk 2]
```diff
@@ -665,97 +665,133 @@ they keep track of what was changed in a file when and by whom,
...
-Regardless of whether a formal tool or a manual approach is chosen,
-the first rule for using version control is that nearly
-everything created by a human being goes under version control as soon as it is created.
+though,
+they will need to coordinate so that only a single person is working on specific files at any time.
+In particular,
+they may wish to create one change log file per contributor,
+and to merge those files whenever a backup copy is made.
+
+Version control tools like Git, Mercurial, and Subversion essentially automate and accelerate
```
Comment (lexnederbragt) [inline index.md]: And then here write something like

'Our second suggested approach is using Version control tools like Git,...'

File: index.md [hunk 3]
```diff
@@ -665,97 +665,133 @@ they keep track of what was changed in a file when and by whom,
...
+the manual process just described:
+
+1.  Instead of requiring users to copy everything into subfolders,
```
Comment (lexnederbragt) [inline index.md:792]: I think the version control parts needs a bit more explanation. It currently assumes the reader knows what it is and how it works. 'Everything is kept in one set of folders without any dates in the names, commit early and commit often, write informative commit messages' are thing that could (should?) be mentioned, before the list of benefits.

**Comment by gvwilson** [comment]
Updated to reflect @lexnederbragt comments - more comments welcome.

**Comment by jennybc** [comment]
This section is looking great!

Just some thoughts/wording that came to mind when reading this. If any of these phrases or metaphors seems useful, feel free to plug in or ask me to do so. Otherwise, just accept the praise.

Re: what we really need from (in)formal VC: [1] assurance you could get back to previous version if really needed to; [2] sane coordination of edits from multiple people.

Re: VC coming from s/w world. And this is sort of why it sucks, esp. at first. Many of us are making ["off-label" use](https://en.wikipedia.org/wiki/Off-label_use) of formal VC systems. 95% of the features aren't needed but clog up the UI and docs. And a few things we need aren't there.

**Pull request merged**